docker-compose.yml

```diff
@@ -1385,9 +1385,8 @@ services:
         tz: ${TZ}
     shm_size: *shm-size
     environment:
-      LIBARROW_DOWNLOAD: "true"
```

Comment:
I know it wasn't part of this PR, but should be bump the R version from `r-rstudio-r-base-4.0-focal-revdepcheck`?

Reply:
I actually did that locally, so yeah :D